src/components/table.js

```diff
@@ -177,25 +178,33 @@ export const FlexTable = ({
     body.current.scrollToPosition({ scrollTop: initialY })
   })
 
-  return div([
+  return div({
+    role: 'table',
+    'aria-rowcount': rowCount,
```

Comment:
Not sure if this is how it's supposed to be, but when I was testing this out locally, I found that this `rowCount` doesn't include the column headers, but the row indices do. Should those match up? It seems inconsistent to have a table report that there are 2 rows, but then have the index of the last row be 3

Reply:
Good catch!